packages/rum/src/domain/record/viewports.spec.ts

```diff
@@ -1,30 +1,57 @@
-import { getScrollX, getScrollY, getWindowWidth, getWindowHeight } from './viewports'
+import { getScrollX, getScrollY, getWindowHeight, getWindowWidth } from './viewports'
 
 function isMobileSafari12() {
   return /iPhone OS 12.* like Mac OS.* Version\/12.* Mobile.*Safari/.test(navigator.userAgent)
 }
 
 describe('layout viewport', () => {
-  beforeEach(() => {
+  const addVerticalScrollBar = () => {
     document.body.style.setProperty('margin-bottom', '2000px')
-  })
+  }
+  const addHorizontalScrollBar = () => {
+    document.body.style.setProperty('margin-bottom', '2000px')
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this function is the same as `addVerticalScrollBar`, shouldn't be `margin-left`?

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏My screen is 2.5k width, could you use a bigger value?